### ISSUE TITLE
Some questions about the fine-tuned models.

### ISSUE THREAD

**Issue by duchenzhuang** [facebookresearch/mae#90]
Thank the authors for making the fine-tuned models available for download. And I tested the models on the Imagenet Val dataset according to the instructions~(in https://github.com/facebookresearch/mae/blob/main/FINETUNE.md).
However, the results I get are slightly different from the results you reported.

1. for ViT-Base, the results I get are *Acc@1 83.746 Acc@5 96.540 loss 0.730. (* Acc@1 83.664 Acc@5 96.530 loss 0.731 in the FINETUNE.md)
2. for ViT-Large, the results I get are * Acc@1 85.954 Acc@5 97.556 loss 0.646. (* Acc@1 85.952 Acc@5 97.570 loss 0.646 in the FINETUNE.md)
3. for ViT-Huge, the results I get are * Acc@1 86.880 Acc@5 98.074 loss 0.585. (* Acc@1 86.928 Acc@5 98.088 loss 0.584 in the FINETUNE.md)

And I also test the models on Imagenet-A~(https://github.com/hendrycks/natural-adv-examples). The results I get are also slightly different from the results you reported.

1. for Vit-B, the result I get on Imagenet-A is 0.364.~(The result in paper is 0.359)
2. for Vit-L, the result I get on Imagenet-A is 0.5627.~(The result in paper is 0.571)
3. for Vit-H, the result I get on Imagenet-A is 0.673.~(The result in paper is 0.682)

I am very grateful to the authors for releasing the code and checkpoints.
I just want to understand how these errors come about. If there is something wrong with my operation or my dataset, I can correct it in time.
